.github/workflows/rust.yml

```diff
@@ -85,3 +85,26 @@ jobs:
         with:
           command: build
           args: --release --verbose --all-features --target x86_64-unknown-linux-gnu
+  
+    build-container:
+    name: Build and Push image to DockerHub.
```

Comment:
indentation seems not correct

Reply:
[comments](https://github.com/ayrat555/el_monitorro/pull/239/commits/9e477ec9d084f45cb6beb4c0146aa478afcdfbdd)